src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -99,7 +98,7 @@ object WorkflowDescriptor {
   }
 
   def apply(id: WorkflowId, sourceFiles: WorkflowSourceFiles, conf: Config): WorkflowDescriptor = {
-    validateWorkflowDescriptor(id, sourceFiles, CromwellServer.backend, conf) match {
+    validateWorkflowDescriptor(id, sourceFiles, CromwellBackend.backend(), conf) match {
```

Comment:
Any reason for the choice of adding in ()? I thought they were considered thoughtcrime in our new, enlightened age